e2e/assertions/compareScreenshot.js

```diff
@@ -57,11 +94,23 @@ exports.assertion = function(componentName, fileName) {
   }
 
   this.pass = function pass(value) {
-    const pass = value <= this.expected
+    let pass = value <= this.expected
 
-    this.message = `Screenshots ${
-      pass ? 'Matched' : 'Match Failed'
-    } for ${fileName} with a tolerance of ${this.expected}%, actual was ${value}%.`
+    if (
+      (updateScreenshots === 'false' && updateAllScreenshots === 'false') ||
+      !promptAccepted ||
+      pass
+    ) {
+      this.message = `Screenshots ${
+        pass ? 'Matched' : 'Match Failed'
+      } for ${fileName} with a tolerance of ${this.expected}%, actual was ${value}%.`
+    } else if (updateScreenshots === 'true') {
+      this.message = 'User updated failing baseline screenshot.'
+      pass = true
+    } else if (updateAllScreenshots === 'true') {
+      this.message = 'Initial screenshot match failed, but baseline screenshot was updated.'
+      pass = true
+    }
```

Comment:
This if...else... block doesn't seem to contain a case for `pass === false`? What happens if screenshots are not being updated and the screenshot failed, or is that not a valid case?